tests/test_models.py

```diff
@@ -392,6 +392,12 @@ class Private(Model):
         self.assertEqual("Field _private of Model class Private defines an explicit Field "
                          "but lacks a type annotation or isn't public", str(ctx.exception))
 
+    def test_mapping_access(self):
+      model = DriedLeaf({'age': 5})
```

Comment:
The body is indented by 6 spaces instead of 8...

Reply:
I had an `.editorconfig` laying around somewhere in parent directories and PyCharm used that.